File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):    
    min_x = 51
    min_y = 51
    max_x = -1
    max_y = -1
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == "#":
                min_x = min(min_x, j)
                min_y = min(min_y, i)
                max_x = max(max_x, j+1)
                max_y = max(max_y, i+1)
    answer = [min_y, min_x, max_y, max_x]
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int lux = 50, luy = 50, rdx = -1, rdy = -1;
        for (int i = 0; i < wallpaper.length; i++) {
            for (int j = 0; j < wallpaper[0].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    lux = Math.min(lux, i);
                    luy = Math.min(luy, j);
                    rdx = Math.max(rdx, i+1);
                    rdy = Math.max(rdy, j+1);
                }
            }
        }
        
        return new int[]{lux, luy, rdx, rdy};
    }
}